util/FirebaseApp.js

```diff
@@ -43,9 +43,25 @@ export function getFirebaseProvider(platform) {
 export async function firebasePlatformSignIn(platform) {
   const provider = getFirebaseProvider(platform);
   const result = await firebase.auth().signInWithPopup(provider);
+  const {
+    displayName,
+    email,
+    emailVerified: isEmailVerified = false,
+    photoURL: avatarURL = false,
+  } = result.user;
   const { accessToken, secret } = result.credential;
   const firebaseIdToken = await firebase.auth().currentUser.getIdToken();
-  return { accessToken, secret, firebaseIdToken };
+  return {
+    displayName,
+    email,
+    isEmailVerified,
```

Comment:
probably dont want to expose `isEmailVerified` for client code

Reply:
Registration form depends on it for toggling email field